packages/node_modules/@webex/redux-module-conversation/src/actions.js

```diff
@@ -433,12 +438,14 @@ export function removeParticipantFromConversation(conversation, participant, spa
  * @returns {function}
  */
 export function handleAdaptiveCardSubmitAction(url, actionInput, parentId, spark) {
-  return (dispatch) =>
+  return (dispatch) => {
```

Comment:
This needs to return a promise, by adding the `{` it will return null. So you need to remove the `{`.